pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -667,18 +667,20 @@ func (dm *DockerManager) runContainer(
 	}
 
 	// Set sysctls if requested
-	sysctls, unsafeSysctls, err := api.SysctlsFromPodAnnotations(pod.Annotations)
-	if err != nil {
-		dm.recorder.Eventf(ref, api.EventTypeWarning, events.FailedToCreateContainer, "Failed to create docker container %q of pod %q with error: %v", container.Name, format.Pod(pod), err)
-		return kubecontainer.ContainerID{}, err
-	}
-	if len(sysctls)+len(unsafeSysctls) > 0 {
-		hc.Sysctls = make(map[string]string, len(sysctls)+len(unsafeSysctls))
-		for _, c := range sysctls {
-			hc.Sysctls[c.Name] = c.Value
+	if container.Name == PodInfraContainerName {
```

Comment:
Sorry for the disappearing for a while. This PR is useful to be merged into a v1.4 patch release. So let's get it through.
Ideally, kubelet should vet all pods for all configured security policies upon start-up, since this scenario will be applicable to other security features too. Restarting pods might be troublesome and so failing pods due to failing security checks might be a good thing to do.

WDYT? We should also validate against the whitelist everytime before applying it in general.